blocks/model.py

```diff
@@ -155,8 +155,12 @@ def __init__(self, outputs):
         for brick in bricks:
             if brick not in children and brick not in self.top_bricks:
                 self.top_bricks.append(brick)
-        if len(set(b.name for b in self.top_bricks)) < len(self.top_bricks):
-            raise ValueError("top bricks with the same name")
+        # check there is no occurence in top bricks name
+        names = Counter([b.name for b in self.top_bricks])
+        if names.most_common(1)[0][1] > 1:
+            repeated_name = names.most_common(1)[0][0]
+            raise ValueError("top bricks with the "
+                             "same name : %s" % repeated_name)
```

Comment:
Please use the new formatting syntax: `"top bricks with the same name: {}".format(name)`